examples/gnrc_minimal/Makefile

```diff
@@ -9,15 +9,6 @@ RIOTBASE ?= $(CURDIR)/../..
 
 BOARD_INSUFFICIENT_MEMORY := chronos msb-430 msb-430h nucleo32-f031
 
-## Uncomment to support the XBee module
```

Comment:
I am opposed to that, I don't think we should have comments about available hw/modules in these makefiles, so if we would keep the comment about the xbee here, why not also have comments about `w5100`, `encxxx`, etc.? IMHO these kind of comments don't belong here.